### DISCUSSION TITLE
DeePMD Training - Multiple Temperatures

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/deepmd-kit#2261] [Q&A]
Hello,

I hope you are doing well.  I would like to know if it is possible to use DeePMD in order to train a model for more than 1 temperature.  For example, let us say we have a DFT training set for Silicon consisting of temperatures at 300, 600, and 900K.  Does this present any issues for the AI model?

Thank you very much!

**Comment by njzjz** [comment]
The potential energy is independent of the temperature, i.e., the potential energy does not change when the temperature changes. It's unclear what you mean when you say you have a DFT training set of multiple temperatures.

**Reply by npiroozan** [reply]
Thank you. Apologies for the lack of clarity. Please allow me to elaborate further. My ultimate goal is to be able to use DeePMD to perform melting point calculations on a particular substance. Before attempting this, I wanted to train a model with 3 sets of input data.  One for each of the temperatures I specified.

With this intermediate setup, then calculate the RDF for the three temperatures accurately with one model.

Is this a realistic aspiration as far as the neural network os concerned?

**Reply by njzjz** [reply]
If you mean the NVT MD simulations sampled under different temperatures, it makes sense. The temperature is just a property of the NVT MD but not DFT. When do MD simulations, the structures are affected by the temperature according to the Boltzmann distribution. DFT energies are not different under different temperatures, thus a potential energy model should work for any temperature if the training data is enough. And also, instead of a model for a temperature, you should say a model for **the simulation under** a temperature.

**Reply by njzjz** [reply]
Now assuming you have understood that the difference is just the structures. Of course, there is no theoretical challenge for a model to learn different structures.

**Reply by npiroozan** [reply]
Understood, thank you so much.

**Reply by npiroozan** [reply]
Hello,

I hope you are doing well.

I am trying to develop input data from Quantum Espresso for Si-Si single pairwise interaction.  How many atoms in the DFT simulation would you recommend to be sufficient in order to be used for training a model to standard levels of accuracy for energy and force?

For example, would 64 atoms in a fcc lattice be sufficient, with increased k-space?

Thank you very much.

**Reply by wanghan-iapcm** [reply]
Highly system dependent. 

I would start by preparing training data from relatively small systems (they are relatively cheap in dft calculations).  Test the model to see if it is able to generalize to larger systems. If yes stop, else add data with larger systems to the training dataset and test the model again. Loop the procedure until converge.

**Reply by lnnbig** [reply]
> If you mean the NVT MD simulations sampled under different temperatures, it makes sense. The temperature is just a property of the NVT MD but not DFT. When do MD simulations, the structures are affected by the temperature according to the Boltzmann distribution. DFT energies are not different under different temperatures, thus a potential energy model should work for any temperature if the training data is enough. And also, instead of a model for a temperature, you should say a model for **the simulation under** a temperature.

Should electronic temperature be a big concern?

**Reply by wanghan-iapcm** [reply]
For temperatures lower than 1eV, one can safely use zero-temperature dft energy surface.
